packages/components/select/src/select.vue

```diff
@@ -30,13 +30,13 @@
           @mouseleave="inputHovering = false"
         >
           <div
-            v-if="multiple"
+            v-if="multiple || isIOS"
             ref="tags"
             :class="nsSelect.e('tags')"
             :style="selectTagsStyle"
           >
             <span
-              v-if="collapseTags && selected.length"
+              v-if="collapseTags && selected.length && !isIOS"
```

Comment:
Same here, you were just removing the selected tag 🤔 I suspect that you were barking up the wrong tree.